reason-react-native/src/apis/Clipboard.md

```diff
@@ -1,14 +1,24 @@
 ---
 id: apis/Clipboard
 title: Clipboard
-wip: true
+officialDoc: https://facebook.github.io/react-native/docs/clipboard
```

Comment:
@sgny do you know that officialDoc is optional & otherwise generated from the module name?